src/components/external/DisputeManagement/components/DefendDisputeFlow/DefendDisputeFlow.scss

```
@@ -107,6 +107,19 @@
         color: style.token(color-label-secondary);
         font-size: style.token(text-caption-font-size);
     }
+
+    &__dropdown-list {
+        max-height: 270px;
+        overflow: scroll;
```

Since we are constraining just the height here, I'd say we explicitly use `overflow-y` instead of `overflow`. Or as an alternative, set the two axes overflow values using the shorthand syntax.

```suggestion
        overflow: hidden scroll;
```